src/writer.js

```diff
@@ -6,12 +6,14 @@ const format = require('./format')
 const encode = require('./encode')
 const tracerVersion = require('../lib/version')
 
+const MAX_SIZE = 8 * 1024 * 1024 // 8MB
```

Comment:
5mb is what "everyone" else does (Python and Go)

Reply:
I think 2MB is enough room for headers etc.